ses/domain/metrics.tf

```diff
@@ -0,0 +1,85 @@
+locals {
+  metrics = {
+    emails     = module.metrics_count.out_map.sent
+    rejections = module.metrics_count.out_map.rejections
+    bounces    = module.metrics_count.out_map.bounces
+    spam       = module.metrics_count.out_map.spam
+    deliveries = module.metrics_count.out_map.deliveries
+    opens      = module.metrics_count.out_map.opens
+    clicks     = module.metrics_count.out_map.clicks
+
+    rejections_percentage = module.metrics_percentage.out_map.rejections
+    bounces_percentage    = module.metrics_percentage.out_map.bounces
+    deliveries_percentage = module.metrics_percentage.out_map.deliveries
+
+    account_bounce_rate = module.metrics_account_reputation.out_map.bounce
+    account_spam_rate   = module.metrics_account_reputation.out_map.spam
+  }
+}
+
+module "cloudwatch_consts" {
+  source = "./../../cloudwatch/consts"
+}
+
+locals {
+  colors = module.cloudwatch_consts.colors
+}
+
+locals {
+  domain_dimensions = {
+    "ses:from-domain" = local.domain
+  }
+
+  metrics_count = {
+    sent       = { name = "Send", label = "Sent", color = local.colors.grey }
```

Comment:
Too few colors ;) I understand that this is not a problem - they will not be presented in the same widget?

Reply:
I'm using `grey` to signify the desired/expected values, in this case `sent` is the desirect value for the `delivered` metric, so that's the logic behind making `sent` grey